### ISSUE TITLE
Make $$. isTargetToShow public

### ISSUE THREAD

**Issue by Hanterdro** [naver/billboard.js#115]
There are public methods to `show`, `hide` and `toggle` targets but no getter for the current visibility status.

Please add a `isShown` method.

I think you just have to make `$$. isTargetToShow` public.

This is e.g. useful for an external legend, to get the initial visibility status of the data.

**Comment by netil** [comment]
IMO, it can be used [.data.shown()](https://naver.github.io/billboard.js/release/latest/doc/Chart.html#data:shown) for that purposes.

```js
let chart = bb.generate({
	data: {
	    columns: [
		    ["data1", 30, 200, 100, 400, 150, 250],
		    ["data2", 10, 20, 10, 40, 15, 25]
	    ]
	}
});

chart.hide("data1");

chart.data.shown();  // or chart.data.shown("data2");
// returns: [{id: "data2", id_org: "data2", values: Array(6)}]

chart.data.shown("data1");
// returns: []
```

**Comment by Hanterdro** [comment]
Yeah, something `!!chart.data.shown().filter(value => (value.id==='data1')).length` works for sure, but it sucks :-)

**Comment by netil** [comment]
In somehow, I agree with your opinion. It should be more straight forward, just returning boolean, instead of array.

But, adding a similar new API also doesn't look nice.
If don't need to be explicitly boolean, just for the condition evaluating length is enough.

```js
if (chart.data.shown("data1").length) {
    // data1 is shown
} else {
   // data1 is hidden
}
```

Anyway, we'll take considering on your opinion for future release.
Thanks sharing your thoughts!
